docs/_pages/releases.md

```diff
@@ -10,6 +10,7 @@ sidebar:
 ## Unreleased
 
 ### What's New
+* Adding `ThatAreAsync()` and `ThatAreNotAsync()` to `MethodInfoSelector` [#645](https://github.com/fluentassertions/fluentassertions/pull/1725)
```

Comment:
`[#645]` -> `[#1725]`
separating `-` between description and link

Reply:
Sorry, I'm not sure I understand this correctly. Do you want 2 changes? Changing the number from 645 to 1725 and adding a `-` between description and link?

Reply:
Yes 👍 
I'll try to be less note-form in my review.

Reply:
I'm unsure if you normally link to an issue or always to the PR, but I've done the change you requested.

Reply:
You could argue for both, but as I know all links in the Fluent Assertions release notes are to PRs.

Reply:
👍